app/controllers/admin_controller.rb

```diff
@@ -102,4 +104,29 @@ def culled_games
   def about
   end
 
+  def stats_display
+    games = Event.current.top_games()
+    titles = Game.where(culled: false).size
+    active_checkouts = Checkout.where(event: Event.current, closed: false).size
+    checked_out = number_to_percentage(active_checkouts.to_f / titles, precision: 0)
+    titles_played = Checkout.where(event: Event.current).joins(game: [:title]).group(:title).count.count
+    total_checkouts = Checkout.where(event: Event.current).size
+    longest_checkout = Checkout.longest_checkout_game_today(Event.current.utc_offset)
+    hours_played = (Checkout.where(event: Event.current).map {|c| c.return_time ? c.return_time.to_f - c.check_out_time.to_f : DateTime.now.to_f - c.check_out_time.to_f }.reduce(:+) / 3600).to_i
+    render json: {
+      current_event: Event.current.short_name,
```

Comment:
Please assign `Event.current` to a variable since you're using so much. ActiveRecord should be caching it, but you should still try to assign repeated values to variables as a best practice.